### PR TITLE
[webgpu] Enable bias and weights for fusedConv2D

### PR DESCRIPTION
**Pull request by axinging**
FIX https://github.com/tensorflow/tfjs/issues/2660

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/2665)

### CODE REVIEW

**Comment by axinging** [comment]
@annxingyuan

**Comment by axinging** [comment]
@annxingyuan  PTAL.
@qjia7 @haoyunfeix @xhcao

File: tfjs-backend-webgpu/src/setup_test.ts [hunk 1]
```diff
@@ -126,6 +126,16 @@ const TEST_FILTERS: TestFilter[] = [
       'fused',     // Not yet implemented.
     ]
   },
+  {
+    include: 'fused conv2d',
+    excludes: [
+      'im2row with prelu',                         // Actual != expected.
```
Comment (annxingyuan) [inline tfjs-backend-webgpu/src/setup_test.ts]: Hm, how big is the error? I'm surprised that these tests needed to be excluded.

Comment (axinging) [reply tfjs-backend-webgpu/src/setup_test.ts]: The reported error is like this on Mac:
Arrays differ: actual[7] = 0, expected[7] = -2.5.

**Pull request merged**